examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/demo_io.c

```diff
@@ -0,0 +1,358 @@
+#include <assert.h>
```

Comment:
the same here if this demo_io.c is not modified, then use it from the mems SDK as a link.